app/controllers/dashboard_controller.rb

```diff
@@ -25,6 +25,20 @@ def show
 
     @climate_actions = ClimateAction.all
     @climate_actions_categories = ClimateActionCategory.all
+
+    # Functions get climate actions
+    @get_all_user_climate_actions = ClimateAction.joins(:user_climate_actions)
+    .where(["user_climate_actions.user_id = ? and climate_actions.id = user_climate_actions.climate_action_id", current_user.id])
+    .select(:name, :description, :status, :user_id)
+
+    @get_all_climate_actions_that_does_not_contain_user_actions = ClimateAction.where.not(id: UserClimateAction.where(user_id: current_user.id))
```

Comment:
Took a while to understand what these two do... If i understand correctly the first one is all user-actions no matter if the status is true or false? And the second one is the rest. Maybe clarify this in the comment above and perhaps a different naming for the second one, e.g. @get_all_climate_actions_without_user_relation